conftest.py

```diff
@@ -57,13 +58,22 @@ def software():
             )
 
 
+@pytest.fixture(autouse=True)
+def dask_config(software):
+    with dask.config.set(
+        {
+            "coiled.account": "dask-engineering",
+            "coiled.software": software,
+        }
+    ):
+        yield
+
+
 @pytest.fixture(scope="module")
-def small_cluster(software, request):
+def small_cluster(request):
```

Comment:
Moving `dask.config.set(...)` into an `autouse` fixture is causing us to no longer use `account="dask-engineering"` in this fixture and are running into errors due to us hitting Coiled CPU limits 

```suggestion
def small_cluster(dask_config, request):
```

Reply:
Thanks, I'll change this